central/complianceoperator/v2/scanconfigurations/service/convert.go

```diff
@@ -248,6 +248,68 @@ func getLatestBindingError(status *storage.ComplianceOperatorStatus) string {
 	return ""
 }
 
+func convertStorageReportDataToV2ScanStatus(ctx context.Context, reportData *storage.ComplianceOperatorReportData, notifierDS notifierDS.DataStore) (*v2.ComplianceScanConfigurationStatus, error) {
```

Comment:
nit:  line breaks might help here too.

Reply:
Done in [37708f8](https://github.com/stackrox/stackrox/pull/13804/commits/37708f8edb6c5dca6d40463f28dc981edceeec49)